TestAssets/TestPackages/dotnet-desktop-binding-redirects/dotnet-desktop-binding-redirects.csproj

```diff
@@ -0,0 +1,43 @@
+﻿<Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props" />
+  
+  <PropertyGroup>
+    <VersionPrefix>1.0.0-rc</VersionPrefix>
+    <TargetFramework>net451</TargetFramework>
+    <AssemblyName>dotnet-desktop-binding-redirects</AssemblyName>
+    <OutputType>Exe</OutputType>
+    <RuntimeIdentifier>win7-x86</RuntimeIdentifier>
```

Comment:
i don't think this supports plural, x86 will work also on x64 and the test is testing binding redirects so that's not too important to support x64